installation/docker.md

```diff
@@ -253,6 +253,89 @@ It then performs all the same steps that the upgrade script and which are perfor
 
 ## Troubleshooting
 
+### Universal Plug and Play (UPnP)
+
+Some bindings, like e.g. [SONOS](https://www.openhab.org/addons/bindings/sonos/), depend on the common [UPnP](https://en.wikipedia.org/wiki/Universal_Plug_and_Play) communication infrastructure provided by openHAB. The protocol is based on IP multicast messages, which is limited to a local subnet[^multicastrouting]. In case you have multiple network adapters in your system (which is likely if you use docker) oepnhab should use `network_mode: host`. It is then necessary to inform openHAB what interface/network shall be used for UPnP discovery.
+
+The network interface to be used can be specified via the `EXTRA_JAVA_OPTS` [Environment Variable](#environment-variables):
+
+```bash
+EXTRA_JAVA_OPTS="-Dorg.jupnp.network.useInterfaces=eno1"
```

Comment:
Maybe I am wrong but I have in mind that suggestions on the community forum also include a second parameter with an IP address ?

Reply:
Yes, and I just added a comment 1 minute ago :wink: 
https://community.openhab.org/t/sonos-in-docker/150660/25

Reply:
I do not know if this might also depend on the settings of openHAB itself (setting in the UI to what IP it binds). To be on the safe side I'll add this parameter here as well. Better be safe than sorry 🙂.